pandas/tests/series/test_arithmetic.py

```diff
@@ -66,6 +66,14 @@ def test_add_series_with_period_index(self):
         with pytest.raises(IncompatibleFrequency, match=msg):
             ts + ts.asfreq("D", how="end")
 
+    def test_string_addition(self):
+        # GH28658
+        a = Series(["hello", "world"])
+
+        result = a + "!"
```

Comment:
We want to test `+ "m"` as well as the issue notes.

Reply:
ok, I've added that, but this branch is messier than I'd like it to be so I'm going to close this PR and clean up this branch and recreate the PR.